security.md

```diff
@@ -0,0 +1,21 @@
+# Reporting Security Issues
+
+Please do not report security vulnerabilities through public GitHub issues.
+Instead, please report them via an email to info@nvaccess.org.
+
+You should receive an acknowledgement email response within 3 business days.
+If for some reason you do not, please follow up via email to ensure we received your original message. 
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue.
+This information will help us triage your report more quickly.
+
+* Type of issue (e.g. denial of service, privilege escalation, e.t.c.)

```

Comment:
I have never seen e.t.c. written this way; etc. seems much more common. Native English speaker, please confirm.

```suggestion
* Type of issue (e.g. denial of service, privilege escalation, etc.)
```

Reply:
Checked the Wikipedia page, `etc.` is the most common form. `e.t.c` is wrong. Thanks